manifests/installem_agent.pp

```diff
@@ -242,10 +243,15 @@
                       Oradb::Utils::Dborainst["em agent orainst ${version}"],],
       }
 
+      $param_ignore_prereq=''
+      if ( $ignore_sys_prerequisite ) {
+        $param_ignore_prereq='-ignorePrereqs'
```

Comment:
Does this actually work?
https://puppet.com/docs/puppet/6.0/lang_variables.html#no-reassignment

Reply:
oops, i'm having doubt......

Reply:
should have use else instead....

Reply:
indeed, good find

Reply:
sorry about that...